migration/1668437287846-removeUnusedBuyCryptoCols.js

```diff
@@ -0,0 +1,33 @@
+const { MigrationInterface, QueryRunner } = require("typeorm");
+
+module.exports = class removeUnusedBuyCryptoCols1668437287846 {
+    name = 'removeUnusedBuyCryptoCols1668437287846'
+
+    async up(queryRunner) {
+        await queryRunner.query(`ALTER TABLE "dbo"."buy_crypto_fee" DROP CONSTRAINT "FK_88516904512221a608f408dadaf"`);
+        await queryRunner.query(`ALTER TABLE "dbo"."buy_crypto_batch" DROP COLUMN "outputReferenceAsset"`);
+        await queryRunner.query(`ALTER TABLE "dbo"."buy_crypto_batch" DROP COLUMN "outputAsset"`);
+        await queryRunner.query(`ALTER TABLE "dbo"."buy_crypto" DROP COLUMN "outputReferenceAsset"`);
+        await queryRunner.query(`ALTER TABLE "dbo"."buy_crypto" DROP COLUMN "outputAsset"`);
+        await queryRunner.query(`ALTER TABLE "dbo"."liquidity_order" DROP COLUMN "referenceAsset"`);
+        await queryRunner.query(`ALTER TABLE "dbo"."liquidity_order" DROP COLUMN "swapAsset"`);
+        await queryRunner.query(`ALTER TABLE "dbo"."deposit_route" DROP CONSTRAINT "FK_416688df9433a5db75efe220cc1"`);
+        await queryRunner.query(`ALTER TABLE "dbo"."deposit_route" ALTER COLUMN "userId" int`);
+        await queryRunner.query(`ALTER TABLE "dbo"."deposit_route" ADD CONSTRAINT "FK_416688df9433a5db75efe220cc1" FOREIGN KEY ("userId") REFERENCES "user"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`);
```

Comment:
The change on deposit_route is not wanted. It's weird that these changes got picked up only now 🤔  we need to add a `{ nullable: false }` at the `user` field of ´CryptoRoute´ and re-run the migration

Reply:
Looks better now :+1: